router/router_server.go

```diff
@@ -130,19 +129,26 @@ func postServerCommands(c *gin.Context) {
 	c.Status(http.StatusNoContent)
 }
 
-// Updates information about a server internally.
+// Updates information about a server internally. This endpoint is deprecated
+// and should not be used. Right now it just returns an immediate response to
+// the caller, and then triggers a background task to then re-sync the server
+// from the Panel.
+//
+// This is handled as it is now in order to avoid a breaking API change, even
+// though internally this is a whole new endpoint.
 func patchServer(c *gin.Context) {
 	s := ExtractServer(c)
 
-	buf := bytes.Buffer{}
-	buf.ReadFrom(c.Request.Body)
-
-	if err := s.UpdateDataStructure(buf.Bytes()); err != nil {
-		NewServerError(err, s).Abort(c)
-		return
-	}
-
-	s.SyncWithEnvironment()
+	go func(s *server.Server) {
+		// Give the Panel a few seconds to make sure the data is properly persisted to
+		// the database after this endpoint returns a successful response.
+		time.AfterFunc(time.Second * 2, func() {
```

Comment:
The only problem I see with this approach is not knowing if wings actually updated the server's environment.  The panel should persist settings regardless if wings is reachable.  It's extremely annoying when I want to update a server (or even a node) and it errors because wings is unreachable.

The settings should always be persisted, then send a request to wings which can fail; not the other way around.

Reply:
Again, this is working around existing functionality to avoid a breaking change on people.

I'd rather rip this out in V2 and leave it as is, even if it is disgusting right now.

Reply:
After thinking this over more, I'll fix the Panel logic and just roll it as 1.6